.gitmodules

```diff
@@ -1,3 +1,7 @@
 [submodule "app/magento"]
```

Comment:
We should remove magento.  We can't currently bind both to port 80.

Reply:
Done.